Dockerfile

```diff
@@ -35,36 +35,33 @@ ARG IRONIC_SOURCE
 ARG SUSHY_SOURCE
 
 COPY sources /sources/
-
 COPY ${UPPER_CONSTRAINTS_FILE} ironic-packages-list ${PKGS_LIST} ${EXTRA_PKGS_LIST:-$PKGS_LIST} ${PATCH_LIST:-$PKGS_LIST} /tmp/
 COPY prepare-image.sh patch-image.sh configure-nonroot.sh /bin/
-
-RUN prepare-image.sh && \
-  rm -f /bin/prepare-image.sh
-
 COPY scripts/ /bin/
 
+RUN prepare-image.sh && rm -f /bin/prepare-image.sh
+
 # IRONIC #
 COPY --from=ironic-builder /tmp/ipxe/src/bin/undionly.kpxe /tmp/ipxe/src/bin-x86_64-efi/snponly.efi /tftpboot/
 COPY --from=ironic-builder /tmp/esp.img /tmp/uefi_esp.img
 
 COPY ironic-config/ironic.conf.j2 /etc/ironic/
-COPY ironic-config/inspector.ipxe.j2 ironic-config/httpd-ironic-api.conf.j2 ironic-config/ipxe_config.template /tmp/
+COPY ironic-config/inspector.ipxe.j2 ironic-config/httpd-ironic-api.conf.j2  /tmp/
+COPY ironic-config/ipxe_config.template /tmp/ipxe_config.template
 
 # DNSMASQ
-COPY ironic-config/dnsmasq.conf.j2 /etc/
+COPY ironic-config/dnsmasq.conf.j2 /tmp/
```

Comment:
These all configs that go to /tmp should be in same COPY.

Reply:
done